backend/src/main/java/jonaszeihe/ninjin/model/Course.java

```diff
@@ -10,10 +10,11 @@
 @Data
 @AllArgsConstructor
 @NoArgsConstructor
-@Builder
+@Builder(toBuilder = true)
 @Document(collection = "courses")
 public class Course {
     @Id
-    private String name;
-    private String duration;
+    private String courseName;
+    private String courseSize;
+    private String courseDescription;
```

Comment:
Why rename these? The "course"-part of your variables is unnecessary as it is implied by the Classname.
`course.getCourseName` for example reads somewhat redundant doesn't it?